src/components/error-list/error-list.component.tsx

```diff
@@ -81,23 +81,27 @@ export default function ErrorList({ errors = [] }: Options): JSX.Element {
     return 0;
   }
 
-  const renderOneTypeOfErrors = (errorData: ErrorTypes): JSX.Element =>
-    errorData.errors && errorData.errors.length > 0 ? (
-      <FoldingSection name={`${errorData.errorDescription} (${errorData.errors.length})`}>
-        {errorData.errors.sort(compareErrors).map(
-          (error, index): JSX.Element => (
-            <ErrorListItem
-              key={`${error.kind ? error.kind : "0"}_${index}`}
-              error={error}
-              index={index}
-              interactable
-            />
-          )
-        )}
-      </FoldingSection>
-    ) : (
-      <></>
-    );
+  const renderOneTypeOfErrors = (errorData: ErrorTypes): JSX.Element => {
+    if (errorData.errors && errorData.errors.length > 0) {
+      const sortedErrors = errorData.errors.sort(compareErrors);
```

Comment:
Chyba lepiej było by poprawić to przez destrukcję.
Bo problem z tego co widzę, jest z tym, że `sort` sortuje w miejscu.
![image](https://user-images.githubusercontent.com/21079319/120121833-f9d23680-c1a5-11eb-87d4-c1984d5affc2.png)

Reply:
ok